spec/features/question_validations/date_question_spec.rb

```diff
@@ -79,6 +79,13 @@
         expect_no_errors
       end
 
+      scenario 'saving with zero values for hour and minute' do
+        fill_in_question('v_date_hour',  '1')
```

Comment:
00:00 is ook een geldige waarde, maar deze faalde eerst in Javascript omdat 0 falsey is.